service/k8sclient/k8client.go

```diff
@@ -313,6 +335,22 @@ func (c *K8Client) DeleteXtraDBCluster(ctx context.Context, name string) error {
 	return c.kubeCtl.Delete(ctx, res)
 }
 
+// GetXtraDBCluster returns an XtraDB cluster credentials.
+func (c *K8Client) GetXtraDBCluster(ctx context.Context, name string) (*XtraDBCredentials, error) {
+	var cluster pxc.PerconaXtraDBCluster
+
+	err := c.kubeCtl.Get(ctx, string(perconaXtraDBClusterKind), name, &cluster)
+	if err != nil {
+		return nil, errors.Wrap(err, "cannot get XtraDb cluster")
+	}
+
+	credentials := &XtraDBCredentials{
+		Host: cluster.Status.Host,
```

Comment:
let's return all values like we do for PSMDB.

Reply:
There are no other credentials to return for the cluster.

Reply:
we can return static values from pmm-managed.